inc/saiacl.h

```diff
@@ -543,6 +567,24 @@ typedef enum _sai_acl_table_attr_t
      */
     SAI_ACL_TABLE_ATTR_FIELD_DST_IPV6,
 
+    /**
+     * @brief Dst IPv6 Address Higher 64 bits
+     *
+     * @type bool
+     * @flags CREATE_ONLY
+     * @default false
+     */
+    SAI_ACL_TABLE_ATTR_FIELD_DST_IPV6_HIGH,
```

Comment:
As agreed, let us break this into 32b words, since there are usecases for lower LSB 32bits:

SAI_ACL_TABLE_ATTR_FIELD_DST_IPV6_HIGH_WORD0
SAI_ACL_TABLE_ATTR_FIELD_DST_IPV6_HIGH_WORD1
SAI_ACL_TABLE_ATTR_FIELD_DST_IPV6_LOW_WORD0
SAI_ACL_TABLE_ATTR_FIELD_DST_IPV6_LOW_WORD1

SAI_ACL_TABLE_ATTR_FIELD_SRC_IPV6_HIGH_WORD0
SAI_ACL_TABLE_ATTR_FIELD_SRC_IPV6_HIGH_WORD1
SAI_ACL_TABLE_ATTR_FIELD_SRC_IPV6_LOW_WORD0
SAI_ACL_TABLE_ATTR_FIELD_SRC_IPV6_LOW_WORD1

Reply:
Thanks Ravi, review comments are incorporated.